Libs/RaptureWebServlet/src/main/java/rapture/server/web/servlet/BaseReflexScriptPageServlet.java

```diff
@@ -165,14 +170,32 @@ void process(String script, Map<String, Object> parameterMap, HttpServletRequest
         try {
             context = BaseDispatcher.validateSession(req);
             if (context != null) {
-                logger.trace("Got session context " + context.debug());
-                kScript.setCallingContext(context);
-                handler.setScriptApi(kScript);
-
-                ReflexExecutor.runReflexProgram(script, handler, masterParameterMap);
-                resp.setCharacterEncoding("UTF-8");
-                resp.getWriter().append(handler.getOutput());
-                resp.setContentType("text/plain");
+                if (context.getValid()) {
```

Comment:
RAP-4166: BaseReflexScriptPageServlet does not check if calling context is valid